src/garth/telemetry.py

```
@@ -36,19 +51,22 @@
 
 def sanitize(text: str) -> str:
     """Sanitize sensitive data from request/response text."""
-    # Sanitize query params (key=value&...)
-    for key in QUERY_PARAM_PATTERNS:
-        text = re.sub(
-            key + r"=[^&\s]*", f"{key}={REDACTED}", text, flags=re.IGNORECASE
-        )
+    # Sanitize query params using pre-compiled patterns
+    for pattern, replacement in zip(
+        _QUERY_PARAM_PATTERNS, _QUERY_PARAM_REPLACEMENTS
+    ):
```

Ruff (0.14.10)

`zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)